### PR TITLE
Added Tests to Unit & Manufacturer 

### PR DESCRIPTION
**Pull request by Vladacdc**

### CODE REVIEW

**Review by andriyshevchenko — DISMISSED** [review]

**Review by VitaliiMaksymiv — DISMISSED** [review]

File: TransIT.BLL/Services/ImplementedServices/ManufacturerService.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            await _unitOfWork.ManufacturerRepository.RemoveAsync(id);
+            await _unitOfWork.SaveAsync();
+        }
+
+        public async Task<IEnumerable<ManufacturerDTO>> SearchAsync(string search)
+        {
+            var manufacturers = await _unitOfWork.ManufacturerRepository.SearchExpressionAsync(
+                search
+                    .Split(new[] { ' ', ',', '.' }, StringSplitOptions.RemoveEmptyEntries)
+                    .Select(x => x.Trim().ToUpperInvariant())
```
Comment (igolovchuk) [inline TransIT.BLL/Services/ImplementedServices/ManufacturerService.cs]: use SearchTokenCollection class as searchexpressionasync input after pull request #50 will be merged

File: TransIT.BLL/Services/ImplementedServices/UnitService.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        {
+            await _unitOfWork.UnitRepository.RemoveAsync(id);
+            await _unitOfWork.SaveAsync();
+        }
+
+        public async Task<IEnumerable<UnitDTO>> SearchAsync(string search)
+        {
+            var units = await _unitOfWork.UnitRepository.SearchExpressionAsync(
+                search
+                    .Split(new[] { ' ', ',', '.' }, StringSplitOptions.RemoveEmptyEntries)
```
Comment (igolovchuk) [inline TransIT.BLL/Services/ImplementedServices/UnitService.cs]: use SearchTokenCollection class as searchexpressionasync input after pull request #50 will be merged

**Review by igolovchuk — CHANGES_REQUESTED** [review]
please make changes described in comments

**Comment by Vladacdc** [comment]
> 
> 
> Manufacturer Of Work?

Ups, Ctrl+H ))

**Review by andriyshevchenko — APPROVED** [review]

**Review by igolovchuk — APPROVED** [review]

**Review by andriyshevchenko — APPROVED** [review]

**Review by BohdanBuhrii — APPROVED** [review]

**Pull request merged**